app_dart/lib/src/request_handling/static_file_handler.dart

```diff
@@ -0,0 +1,145 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:appengine/appengine.dart';
+import 'package:meta/meta.dart';
+import 'package:mime/mime.dart';
+
+import 'exceptions.dart';
+
+/// A class based on [RequestHandler] for serving static files.
+@immutable
+class StaticFileHandler {
+  /// Creates a new [StaticFileHandler].
+  const StaticFileHandler();
+
+  /// The location for the Flutter application
+  // TODO(chillers): Remove this when deployed for production use. https://github.com/flutter/cocoon/issues/472
+  static const String flutterBetaUrlPrefix = '/v2';
```

Comment:
This shouldn't be hard-coded in the handler -- the handler should take the info it needs to convert from request path to static file in its constructor, then that info should be passed to the handler from `bin/server.dart`

Reply:
Added